tox.ini

```diff
@@ -25,7 +25,9 @@ setenv =
     clocale: LC_ALL = C
     devdeps: PIP_VERBOSE = 1
     devdeps: PIP_EXTRA_INDEX_URL = https://pypi.anaconda.org/liberfa/simple https://pypi.anaconda.org/scientific-python-nightly-wheels/simple
+    devdeps: UV_EXTRA_INDEX_URL = https://pypi.anaconda.org/liberfa/simple https://pypi.anaconda.org/scientific-python-nightly-wheels/simple
     mpldev: PIP_EXTRA_INDEX_URL = https://pypi.anaconda.org/scientific-python-nightly-wheels/simple
+    mpldev: UV_EXTRA_INDEX_URL = https://pypi.anaconda.org/scientific-python-nightly-wheels/simple
```

Comment:
Why do we need `UV_EXTRA_INDEX_URL` that would only affect dev jobs, not oldestdeps?

Reply:
Because of how tox-uv is enabled by mere installation, so, even if it doesn't happen in CI, this is needed to allow running dev deps jobs locally if tow-uv is installed.